flutter/lib/src/sentry_native_channel.dart

```diff
@@ -28,7 +28,7 @@ class SentryNativeChannel {
 
   Future<void> beginNativeFrames() async {
     try {
-      await _channel.invokeMapMethod<String, dynamic>('beginNativeFrames');
+      await _channel.invokeMethod('beginNativeFrames');
```

Comment:
This should break the test `beginNativeFrames` in sentry native channel test

Reply:
I guess we oversaw this when refactoring. We are not reporting back any values when calling `beginNativeFrames` on the iOS plugin: https://github.com/getsentry/sentry-dart/blob/5d15a51e3c8c0d5af29d48868ee07f994610b274/flutter/ios/Classes/SentryFlutterPluginApple.swift#L433

Reply:
The idea may have been to call this to get the initial frames and calculate a diff when ending the collection with `endNativeFrames`. Do you thinkt the current implementation is working as expected? 🤔

Reply:
@denrase double check the review and comments, I guess it's intentional because iOS does the delta calculation already? don't recall either.

Reply:
Ah yea, but still we have "dead/useless" code in the plugin, as we are not returning values. Think i forgot this.

Reply:
think we can leave the swift code for now, as i am not sure if we need to access the value once for init